doc/user/content/sql/create-source/kafka.md

```diff
@@ -304,6 +304,48 @@ As long as any offset continues increasing, Materialize is consuming data from
 the upstream Kafka broker. For more details on monitoring source ingestion
 progress and debugging related issues, see [Troubleshooting](/ops/troubleshooting/).
 
+### Consumer groups
+
+When Materialize consumes from a Kafka topic for a source, it uses a consumer group whose ID contains
+the ID of your Materialize region, as well as the IDs of the involved source and connection.
+
+To determine the consumer group ID for a source with a specified name, you can use the following
+query:
+
+```sql
+SELECT
+  group_id_base
+FROM mz_internal.mz_kafka_sources ks
+JOIN mz_sources s
+ON
+  s.id = ks.id
+WHERE
+  s.name = '<src_name>'
+```
+
+Note that its called a `group_id_base`, instead of `group_id`. This is because in the future, this ID may be
+enriched with additional information about individual replicas. Because of this, `mz_kafka_sources` is currently
+in the [`mz_internal`](/sql/system-catalog/mz_internal/) schema.
+
+Note that the returned `group_id_base` describes the *prefix* of the consumer group ID that Materialize will use.
+Materialize may use multiple consumer groups to consume from the source. Each consumer group will append a
+distinct suffix to the `group_id_base`.
+
+You should not make assumptions about the number of consumer groups that Materialize will use to consume from a
+given source. The only guarantee is that the ID of each consumer group will begin with `group_id_base`.
+
+### Committing offsets
+
+Materialize commits offsets as it consumes and records Kafka topics. Committed offsets are associated with the
+[consumer group(s)](#consumer-groups) Materialize uses for each source, and can be used to monitor lag on the
+Kafka topic.
+
+Note that committed offsets:
+- Are only for monitoring, Materialize tracks its own progress internally. See [Monitoring source progress](#monitoring-source-progress)
+  for more information.
+- Materialize does not use the group management protocol, assigning partitions manually. Tools that expect the
```

Comment:
"Tools that expect the group to contain active members will be disappointed" is not polished enough phrasing for our user-facing docs.